test1.js

```diff
@@ -1,5 +1,5 @@
 var foo = '' ;
 
-if(foo  === "bar") {
+if(foo  === 'bar') {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26941)
```suggestion
if(foo  === "bar") {
```